File: guanabara-java/src/com/danillo/algoritmos/lista10/Exercicio100.java
package com.danillo.algoritmos.lista10;

import java.util.Scanner;

public class Exercicio100 {
    private static float media(float n1, float n2){
        return (n1 + n2)/2;
    }

    private static String situacao(float media){
        if(media >= 7){
            return "APROVADO";
        }else if(media >= 5){
            return "RECUPERAÇÃO";
        }else{
            return "REPROVADO";
        }
    }

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira a primeira nota: ");
        float n1 = entrada.nextFloat();

        System.out.println("Insira a segunda nota: ");
        float n2 = entrada.nextFloat();

        float media = media(n1, n2);
        System.out.println("Média: " + media);

        String situacao = situacao(media);
        System.out.println("Situação: " + situacao);
    }
}

File: guanabara-python/src/lista10/exercicio100.py
def media(a, b):
    return (a + b)/2


def situacao(media):
    if media >= 7:
        return "APROVADO"
    elif media >= 5:
        return "RECUPERAÇÃO"
    else:
        return "REPROVADO"


a = float(input("Insira a primeira nota: "))
b = float(input("Insira a segunda nota: "))
media = media(a, b)
print(media)
situacao = situacao(media)
print(situacao)
